contributing_to_docs/doc_guidelines.adoc

```diff
@@ -257,6 +257,29 @@ link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-mod
 When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syntax to suppress TOC formatting within a module. Do not use `==` headings or xrefs in modules.
 ====
 
+[id="using-conscious-language"]
+== Using conscious language
+
+To assist with the removal of problematic language from the documentation, use the following terminology when referring to OpenShift control plane or compute nodes:
+
+[options="header"]
+|===
+|Branch |Control plane node reference |Compute node reference
+
+|`main` and `enterprise-4.9`
+|Control plane node
+|Compute node
+
+|`enterprise-4.8` and earlier
```

Comment:
I thought that 4.8 and earlier versions were actually being handled a little differently. I thought that the (aka) part was only added to 4.8+. Aren't there still instances of master that exist in 4.7 and 4.6 that are being kept?

Reply:
(aka) part is added to 4.8 and below. Sorry @pneedle-rh my fault for giving you the wrong information on this.

Reply:
Ok, I am not reading this right. :) 

4.9 we exclusively use control plane instead of master. 4.8 and below we use the aka part.

Reply:
@vikram-redhat I thought you only did the sweep to add "(also known as master)" to 4.8+?

Reply:
No. To 4.8, 4.7 and 4.6: https://github.com/openshift/openshift-docs/pull/34407

Reply:
Okay thanks for the clarification on that.